app/views/components/addToList.scala.html

```diff
@@ -15,13 +15,22 @@
             <dd class="hmrc-add-to-a-list__change">
                 <a class="govuk-link" href="@item.changeUrl">
                     <span aria-hidden="true">@messages("site.change")</span>
-                    <span class="govuk-visually-hidden">@messages(changePrefix, item.name)</span>
+                    @if(countryFromName.isDefined) {
+                        <span class="govuk-visually-hidden">@messages(changePrefix, countryFromName.get, item.name)</span>
```

Comment:
Did you check that why pass a different changePrefix through, this is effectively going to be the key that decides which message we use (either the 2 param or 1 param one) we will need to make sure its a different changePrefix in each case